src/common/string/format-list.ts

```diff
@@ -0,0 +1,18 @@
+import { HomeAssistant } from "../../types";
+import "../../resources/intl-polyfill";
+
+export const longConjunctionFormatter = (
+  hass: HomeAssistant
+): Intl.ListFormat =>
```

Comment:
Can we do this the same as we do for our date/time formatters:

https://github.com/home-assistant/frontend/blob/dev/src/common/datetime/format_date.ts#L7-L21